src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/form.html.twig

```diff
@@ -25,75 +25,73 @@
 
 {% import '@PrestaShop/Admin/macros.html.twig' as ps %}
 
-{% block form %}
-    {{ form_start(metaForm) }}
-      <div class="card">
-        <div class="card-header">
-          {{ 'Meta tags'|trans({}, 'Admin.Shopparameters.Feature') }}
-        </div>
-        <div class="card-block row">
-          <div class="card-text">
-            {{ ps.form_group_row(metaForm.page_name, {'attr': {'class': 'js-meta-page-name', 'data-toggle': 'select2', 'data-minimumResultsForSearch': '7'}}, {
-              'label': 'Page name'|trans({}, 'Admin.Shopparameters.Feature'),
-              'help': 'Name of the related page.'|trans({}, 'Admin.Shopparameters.Help')
-            }) }}
+{{ form_start(metaForm) }}
+  <div class="card">
+    <div class="card-header">
+      {{ 'Meta tags'|trans({}, 'Admin.Shopparameters.Feature') }}
+    </div>
+    <div class="card-block row">
+      <div class="card-text">
+        {{ ps.form_group_row(metaForm.page_name, {'attr': {'class': 'js-meta-page-name', 'data-toggle': 'select2', 'data-minimumResultsForSearch': '7'}}, {
```

Comment:
use multiline please, really hard to read

Reply:
Done.

Reply:
Thanks :+1: